wcivf/apps/people/templates/people/includes/_person_multiple_current_candidacies.html

```diff
@@ -0,0 +1,20 @@
+{% load i18n static %}
+
+<h5>{{ person.name }}
```

Comment:
Here you are referencing `person.name` but where you're including the template you are calling it with
```
person_name=object.name
```
Looks like this should probably be `{{ person_name }}`